lib/sse_consumer.ex

```diff
@@ -0,0 +1,150 @@
+defmodule SSEConsumer do
+  require Logger
+  use GenServer
+
+  defmodule Request do
+    @enforce_keys ~w(method url body headers)a
+    defstruct @enforce_keys
+  end
+
+  defmodule State do
+    @moduledoc false
```

Comment:
Are you making use of `moduledoc`s anywhere else

Reply:
I'm about to. 

But even if we don't provide any deliberate docs, you can generate the most basic docs for the code, and only showing the modules relevant to the consumer can be helpful.